apps/src/applab/designMode.js

```diff
@@ -220,10 +225,19 @@ designMode.onDonePropertiesButton = function() {
 };
 
 designMode.onDeletePropertiesButton = function(element, event) {
-  element.parentNode.removeChild(element);
-  if ($(element).hasClass('screen')) {
+  var isScreen = $(element).hasClass('screen');
+  var toRemove = element;
+  var parent = element.parentNode;
+  if ($(parent).is('.ui-resizable')) {
+    toRemove = parent;
+    parent = parent.parentNode;
+  }
+  parent.removeChild(toRemove);
```

Comment:
Nit: jQuery `.remove()` may be a little more readable (you don't need to pass in the parent, just call it on the node you want to remove).